tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -224,11 +224,39 @@ def test_transformer():
         transforms.RandomHue(0.1),
         transforms.RandomLighting(0.1),
         transforms.ToTensor(),
+        transforms.RandomRotation([-10., 10.]),
         transforms.Normalize([0, 0, 0], [1, 1, 1])])
 
     transform(mx.nd.ones((245, 480, 3), dtype='uint8')).wait_to_read()
 
 
+@with_seed()
```

Comment:
For tests, is testing the dimension only enough?

Reply:
I added some tests with dummy images in `test_rotate`